app/off.py

```diff
@@ -78,8 +79,19 @@ def wikidata(query, value):
         OSM_relation = "https://www.openstreetmap.org/relation/{}".format(osm)
     else:
         OSM_relation = ""
-
-    return (
+    Entities = namedtuple(
+        "Entities",
+        [
+            "label_tag",
+            "description_tag",
+            "image_url",
+            "entity_id",
+            "OSM_relation",
+            "INAO_relation",
+            "wikipedia_relation",
+        ],
+    )
```

Comment:
Define it at module level, not in the function. 

And (just to clarify) here capital letter is welcome because namedtuple return you a class :-)

Reply:
just to explain: namedtuple really return a class, it's just a shortcut to define a class with a specific behaviour.
That's why it belongs to module level (except specific use case)